lib/tasks/instrumentation_generator/README.md

```diff
@@ -1,42 +1,63 @@
 # Instrumentation Generator
 
-## Idea
-Create a CLI, similar to Rails’ generators and scaffold scripts, to create the required files and basic classes for instrumenting new libraries. The hope is that the parts of adding new instrumentation that is repetitive can be eliminated and/or reduced so that it takes less time to add new instrumentation and we can spend development time considering the best attributes to collect and interactions to measure. I also hope this project will reduce the overall toil related to adding new instrumentation.
-
-The files we create with little variation besides library and method names are:
-* Lib
-  * Dependency detection
-  * Chain
-  * Prepend
-  * Instrumentation
-* Tests
-  * configuration/newrelic.yml
-  * Envfile
-  * Test file
-
-Furthermore, we also create very similar snippets inside the default source configuration file that could be generated by this project as well.
-
-## Outcome
-A prototype outside the agent has been created that generates the required files to create new instrumentation. This prototype accepts three arguments: name (name of the library), method (method to instrument), args (arguments for the method).
-
-The Ruby gem Thor, a toolkit for building powerful command-line interfaces used in Bundler, Vagrant, Rails and others powers this CLI.
-
 ## Usage
 
 `thor list` can be passed to get a list of all available thor commands
 
 ### instrumentation:scaffold
+
 This task requires one parameter by default: the name of the library or class you are instrumenting. This task generates the basic file structure needed to add new instrumentation to the Ruby agent.
 
+Run it using:
+
+`thor instrumentation:scaffold gem_name`
```

Comment:
How can I obtain this `thor` binary? Should the Thor gem be added as a dev dependency to our .gemspec file?

Reply:
Thor will be added as a dev dependency to our .gemspec file as part of this PR! It's a bit hard to see with the alphabetizing changes. 

See: https://github.com/newrelic/newrelic-ruby-agent/pull/1402/files#diff-1099f92de9acf3d6d134894bd17f60b5be5b96668f0d30a36785fec0c37dae0bR61

Reply:
whoops! I definitely saw and liked the alphabetization change when I first looked over the PR and then forgot about it being present. Sorry for that.